FrEee.WinForms/post-build.bat

```diff
@@ -4,15 +4,15 @@ for /f "tokens=1,* delims= " %%a in ("%*") do set ALL_BUT_FIRST=%%b
 
 echo Copying assets to %ALL_BUT_FIRST%\bin\%1\net472

```

Comment:
Again we are mentioning a nonexistent net472 folder.

Reply:
And fixed again. :)